### PR TITLE
Walletlink (Coinbase wallet) integration

### PR DESCRIPTION
**Pull request by vfn**
Add WalletLink (Coinbase wallet) as a "built-in" provider.

### CODE REVIEW

File: src/providers/connectors/walletlink.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    const walletLink = new WalletLink({
+      appName,
+      appLogoUrl,
+      darkMode
+    });
+
+    try {
+      const provider = walletLink.makeWeb3Provider(rpc, chainId);
+      await provider.enable();
```
Comment (vishnumad) [inline src/providers/connectors/walletlink.ts:37]: `provider.enable()` is deprecated per [EIP-1102](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1102.md). 
```suggestion
      await provider.send('eth_requestAccounts');
```

**Comment by edkek** [comment]
Any updates on this PR? Would love to add WalletLink support to our dapp

**Comment by vfn** [comment]
> Any updates on this PR? Would love to add WalletLink support to our dapp

@edkek while the PR is not merged, you can add a custom integration as I explain in this issue https://github.com/Web3Modal/web3modal/issues/246#issuecomment-796326985

**Comment by xpluscal** [comment]
Any updates on this?

**Pull request merged**